R/getTicks.R

```diff
@@ -23,18 +26,19 @@
 ##' an object of the type selected in \code{returnAs}.
 ##' @author Dirk Eddelbuettel
 getTicks <- function(security,
-                     eventType = "TRADE",
+                     eventType = c("TRADE"),
```

Comment:
This doesn't need `c("TRADE")` as `"TRADE"` is still the same (but I like you made the argument a vector).